pytest-pixl/README.md

```diff
@@ -8,6 +8,11 @@ Installable `pytest` plugin module providing common test fixtures used throughou
 python -m pip install -e ../pixl_core -e ".[test]"
 ```
 
+## pytest-cov’s engine
+```
+COV_CORE_SOURCE=src COV_CORE_CONFIG=.coveragerc COV_CORE_DATAFILE=.coverage.eager pytest --cov=src --cov-append
```

Comment:
do we need to set these variables when running github actions?

Reply:
Well spotted. Just sorted out here [85b7ff7](https://github.com/SAFEHR-data/PIXL/pull/501/commits/85b7ff74e21e68a5d7a48690250716cb400cb9ab)